edvart/report_sections/timeseries_analysis/time_analysis_plot.py

```diff
@@ -115,8 +114,6 @@ def _time_analysis_colored_plot(df, columns=None, color_col=None):
         layout = dict(xaxis_rangeslider_visible=True)
         if not utils.is_categorical(df[color_col]):
             raise ValueError(f"Cannot color by non-categorical column `{color_col}`")
-        if df[color_col].nunique() > 20:
-            warnings.warn("Coloring by categorical column with many unique values!")
```

Comment:
This warning isn't converted to markdown printout but deleted. This should be a separate PR, likely a one merged before this one.

Reply:
Right, removed in this PR.

Reply:
Created a separate PR https://github.com/datamole-ai/edvart/pull/104